stubs/model.py

```diff
@@ -1342,22 +1309,22 @@ def get_block_J(self, Fsum, u):
         for idx, Ji in enumerate(J):
             idx_i, idx_j = divmod(idx, len(u))
             if Ji is None or Ji.empty():
-                fancy_print(
+                logger.warning(
```

Comment:
This occurs whenever there are two non-adjacent compartments, can probably be set to "info" rather than "warning"

Reply:
Fixed in [f8eaa1a](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/f8eaa1aaec9d83148ef6dac5741a4913832d799b)